### PR TITLE
added dept links to docs and work history

### LINKED ISSUE #50: Change work history agency names to hyperlinks.
i.e. Augusta County Sheriff's Office in Work History can be selected, so the user can navigate to the ACSO.
i.e. Under FOIA Library, a file labeled with the Department Augusta County Sheriff's Office links to the Augusta County Sheriff's Office.

### PR DESCRIPTION
**Pull request by bryan-va**
added links to document departments and work history dept
closes #50

### CODE REVIEW

**Comment by kenoshabrickk** [comment]
@bryan-va, this is amazing. This makes the site inexplicably more useful. I really appreciate you managing that even though it's more front-end stuff. I am stoked :)
-a

File: OpenOversight/app/main/views.py [hunk 1]
```diff
@@ -331,6 +331,7 @@ def officer_profile(officer_id):
             .all()
         )
         assignments = Assignment.query.filter_by(officer_id=officer_id).all()
```
Comment (kenoshabrickk) [inline OpenOversight/app/main/views.py:333]: We want the department work history ("Unit") to link from the officer page (what you did is actually a very useful addition, though). 

I'm not editing the code directly cause I have no idea what I'm doing & you do. Would that modification just be changing 'assignments' & 'Assignment' here to the variable for unit or Units?  (Would love to keep what you've got also, though)

File: OpenOversight/app/templates/documents.html
Comment (kenoshabrickk) [inline OpenOversight/app/templates/documents.html:1]: YESSS

File: OpenOversight/app/templates/partials/officer_assignment_history.html
Comment (kenoshabrickk) [inline OpenOversight/app/templates/partials/officer_assignment_history.html:1]: This is super smart & so much cleaner than before

**Review by kenoshabrickk — CHANGES_REQUESTED** [review]
Thank you so much for doing this even though it's more front-end stuff :) This is sooo much cleaner, but I was hoping for links to departments in the "Units" field (for seeing work histories). This is a really valuable addition though for usability, so it works out :) Can we keep these changes but do the same thing for Units?

**Comment by kfinity** [comment]
> Thank you so much for doing this @bryan-va even though it's more front-end stuff :) This is sooo much cleaner, but I was hoping for links to departments in the "Units" field for viewing an officer's work history (my bad, I was totally not clear & I'll add descriptions in the future to issues). This is a really valuable addition though for usability, so it works out :) Can we keep these changes but do the same thing for Units?

I'm not sure I understand what you mean... so like in this screenshot, did you want "Deputy Sheriff" to also be a link to Fairfax County Sheriff's Office officer list?
image

**Pull request merged**